cluster-setup/ci-cluster/performance/performance_profile.patch.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: performance.openshift.io/v1alpha1
 kind: PerformanceProfile
 metadata:
-  name: ci
+  name: performance
```

Comment:
This one is still not merged openshift/machine-config-operator#1356, so can you change it back to a shorter name

Reply:
fixed